setup.py

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env python
+
+# Copyright (c) 2017 Red Hat, Inc. All rights reserved. This copyrighted
+# material is made available to anyone wishing to use, modify, copy, or
+# redistribute it subject to the terms and conditions of the GNU General
+# Public License v.2 or later.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE. See the GNU General Public License for more
+# details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+"""Install skt using setuptools."""
+from setuptools import setup
+
+
+setup()
```

Comment:
I don't know how Python packaging works, sorry, but does this file have to have `.py` in the name?

Reply:
Yes, the `.py` is required because pip/setuptools look for `setup.py` and `setup.cfg` (among other files) when installing a new python module.